src/main/java/net/neoforged/neoforge/client/event/RegisterRenderBuffersEvent.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.event;
+
+import com.mojang.blaze3d.vertex.BufferBuilder;
+import java.util.SortedMap;
+import net.minecraft.client.renderer.RenderType;
+import net.neoforged.bus.api.Event;
+import net.neoforged.fml.event.IModBusEvent;
+import org.jetbrains.annotations.ApiStatus;
+
+public class RegisterRenderBuffersEvent extends Event implements IModBusEvent {
```

Comment:
This event and its methods need javadoc, especially explaining the use case for this event.